graylog2-server/src/main/java/org/graylog2/indexer/counts/Counts.java

```diff
@@ -50,13 +57,22 @@ private long totalCount(final String[] indexNames) {
             return 0L;
         }
 
+        final Count request = new Count.Builder()
+                .query("{\"query\":{\"match_all\":{}}}")
```

Comment:
While this is rather easy for the `Counts` class, do we really want to hardcode static query strings instead of using some helper to build them in general for the long run?